types/hapi/index.d.ts

```diff
@@ -880,16 +884,40 @@ export interface CorsConfigurationObject {
  * [See docs](https://hapijs.com/api/16.1.1#serverextevents)
  * For context see RouteAdditionalConfigurationOptions > ext
  */
-export interface ServerExtConfigurationObject {
+
+// STUFF
+export interface ServerStartExtConfigurationObject {
+    /** the extension point event name. */
+    type: ServerStartExtPoints;
+    /**
+     * a function or an array of functions to be executed at a specified point during request processing. The required extension function signature is see ServerExtFunction or see ServerExtRequestHandler
+     */
+    method: ServerExtFunction
+    options?: ServerExtOptions;
+}
+
+export interface ServerRequestExtConfigurationObject {
     /** the extension point event name. */
-    type: ServerExtPoints;
+    type: ServerRequestExtPointsBase;
     /**
      * a function or an array of functions to be executed at a specified point during request processing. The required extension function signature is see ServerExtFunction or see ServerExtRequestHandler
      */
-    method: ServerExtMethod | ServerExtMethod[];
+    method: ServerExtRequestHandler
```

Comment:
This still needs to be a function or an array of functions.  Same comment for `ServerStartExtConfigurationObject` and `ServerRequestExtConfigurationObjectWithRequest`